shaptools/hana.py

```diff
@@ -701,3 +701,19 @@ def unset_ini_parameter(
             file_name=file_name, layer=layer, layer_name=layer_name,
             set_value=False, reconfig=reconfig, key_name=key_name,
             user_name=user_name, user_password=user_password)
+
+    def extract_hdbserver(self, sar_file, options=None, user=None, password=None, remote_host=None):
```

Comment:
You don't need to pass `user`, `password` and `remote_host` options. They are already part of the class. You can retrieve them using `self` option.

Reply:
And I wonder, if we need these methods. They don't do anything, just call again other method without adding any special thing...
If we can, I would remove them, they add not needed complexity